### PR TITLE
added a blank space before and after the book list

### LINKED ISSUE #8: Add a blank line before and after a list of books
In the ui.py `show_books` function, print a blank line before and after the list of books, or the 'no books' message.

### PR DESCRIPTION
**Pull request by xm6264jz**
@yp0975eu  check out my code. I have added a blank space before and after the book list,

### CODE REVIEW

File: ui.py [hunk 1]
```diff
@@ -25,13 +25,13 @@ def show_books(books):
      :param books: the book list """
 
     # TODO print a blank line before and after the book list or no books message
```
Comment (yp0975eu) [inline ui.py]: This will no longer be a TODO item. The comment on line 27 can be removed.

Comment (xm6264jz) [reply ui.py]: @yp0975eu  I have removed the todo comment

**Comment by yp0975eu** [comment]
I still only see one commit. Did you push your updates?

File: ui.py (hunk 1 above)
Comment (yp0975eu) [reply ui.py]: I still see it.

**Review by yp0975eu — APPROVED** [review]
This looks good

**Pull request merged**